webui/src/App.js

```diff
@@ -145,22 +169,33 @@ function ValueMonitor(props) {
       }
     });
 
-    const render = () => {
-      // ********** Canvas setup **********
-      // This has to be here in case the user resizes the window.
+    const setDimensions = () => {
       // Adjust DPI so that all the edges are smooth during scaling.
       const dpi = window.devicePixelRatio || 1;
-      const style = {
-        height() {
-          return +getComputedStyle(canvas).getPropertyValue('height').slice(0,-2);
-        },
-        width() {
-          return +getComputedStyle(canvas).getPropertyValue('width').slice(0,-2);
-        }
+
+      canvas.width = canvas.clientWidth * dpi;
+      canvas.height = canvas.clientHeight * dpi;
+    };
+
+    setDimensions();
+    window.addEventListener('resize', setDimensions);
+
+    // Add background fill.
+
+    const bodyFontFamily = window.getComputedStyle(document.body).getPropertyValue("font-family");
+    const valueLabel = valueLabelRef.current;
+    const thresholdLabel = thresholdLabelRef.current;
+
+    // cap animation to 60 FPS (with slight leeway because monitor refresh rates are not exact)
+    const minFrameDuration = 1000 / 60.1;
```

Comment:
Can we mention that these durations/timestamps are in ms? Alternatively, rename to minFrameDurationMs or similar.

Reply:
renamed to minFrameDurationMs